.github/workflows/auto-update.yml

```
@@ -0,0 +1,148 @@
+          pnpm openapi:rebuild
+          pnpm build
+          echo "SDK regenerated successfully"
+      
+      - name: Test across Node versions
+        if: steps.check-changes.outputs.has_changes == 'true'
+        uses: ./.github/workflows/test-sdk.yml
+        with:
+          node-versions: '[18, 19, 20, 21, 22]'
+          upload-coverage: 'false'
+      
+  bump-version-and-publish:
```

actionlint (1.7.7)

job "bump-version-and-publish" needs job "test" which does not exist in this workflow

(job-needs)